MBFaker/MBFakerInternet.m

```diff
@@ -28,20 +28,64 @@ + (NSString*)userName {
     
     for (int i=0; i<1+arc4random()%2;i++) {
         if (i == 0)
-            [components addObject:[MBFakerName firstName]];
+            [components addObject:[self fixUmlauts:[MBFakerName firstName]]];
         else
-            [components addObject:[MBFakerName lastName]];
+            [components addObject:[self fixUmlauts:[MBFakerName lastName]]];
```

Comment:
Umlauts were not originally replaced here, but I think it makes sense.  Let me know if you disagree.